### DISCUSSION TITLE
make tags mandatory when uploading

### DISCUSSION THREAD

**Discussion by Eleksar387** [civitai/civitai#209] [Ideas]
Excluding stuff is difficult if people do not even provide tags.
I think it would be good if people are forced to provide, say three general tags which they have to pick from. Like "anime, celebrity, style". Some people do not use tags at all, others use very specific tags which do not help much if you'd like to filter general stuff.

**Comment by JustMaier** [comment]
Yeah. This is definitely an issue. One concern I have about making the tags required is that since it's free-form add you can add whatever tag you want, that it still won't actually help solve the problem, since people might just not tag at the level we actually need to help with categorization.

I've been thinking instead that maybe it makes the most sense to have a "Category" field that is required that has a few curated tags and they can only choose one. What do you think?

**Comment by Eleksar387** [comment]
@JustMaier I am all in for it. Just keep it short and simple. E.g.

- Celebrity
- Anime/Manga
- Style
